src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java

```diff
@@ -27,7 +27,7 @@
   @FindBy(css = ".local-wikia-search a")
   private WebElement searchSuggestion;
 
-  @FindBy(xpath = "//ul[contains(@class, \"local-nav-menu\")]/li/*")
```

Comment:
if something is good practice it doesn't mean that it should be used always.
in this situation your selector returns list element "Menu" while current it is not